cinder/backup/manager.py

```diff
@@ -1065,15 +1149,24 @@ def _attach_volume(self, context, volume, properties):
             return self._connect_device(conn)
         except Exception:
             with excutils.save_and_reraise_exception():
+                trace_utils.trace(
+                    context, trace_utils.ACTION_VOLUME_BACKUP,
+                    volume, "Failed volume_rpcapi.initialize_connection",
+                    failed=True
+                )
                 try:
                     self.volume_rpcapi.terminate_connection(context, volume,
                                                             properties,
                                                             force=True)
                 except Exception:
-                    LOG.warning("Failed to terminate the connection "
-                                "of volume %(volume_id)s, but it is "
-                                "acceptable.",
-                                {'volume_id': volume.id})
+                    msg = ("Failed to terminate the connection "
+                           "of volume %(volume_id)s, but it is "
+                           "acceptable." %
+                           {'volume_id': volume.id})
+                    trace_utils.trace(
+                        context, trace_utils.ACTION_VOLUME_BACKUP,
+                        volume, msg, failed=True
+                    )
```

Comment:
This warning was promoted to error by using `trace_utils`, which makes it more important than before. Maybe, since the message already says that "it is acceptable", it's not a `failed=True` message?

Reply:
removed failed=True.   It will be demoted to an info....:(.

Reply:
Right. `trace_utils` can only replace ERROR and INFO messages without changing the meaning. We have to ensure we take the right decision on which messages should stay in addition to trace_utils (because WARNING is actually necessary) and which should be promoted. This is definitely not an error, because the messages already tells us it's not. What do we do with warnings anyways? Basically ignore them like infos. So the only reason I see , why this might want to be a warning is that it also comes up when the log-level is set to warning i.e. higher than info, because it's an "important info message" O.o

Reply:
I could add an option argument, loglevel, which defaults to info.  If they specify it other than that it will be whatever they want.  failed=True still means ERROR, otherwise the loglevel will be used.

I don't think there are many cases currently where I changed the log to a trace_utils trace that isn't an info or error currently.

Reply:
I'd say either stick with the `failed` arg, or _replace_ it with a `log_level` arg and have INFO and ERROR mean what `failed=False` and `failed=True` mean currently. Don't have both.

Reply:
changed all the failed=True calls to loglevel.